db/data/20211117164147_update_trainee_trns.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class UpdateTraineeTrns < ActiveRecord::Migration[6.1]
+  def up
+    [
+      { trainee_id: "C123414", trn: 2073539 },
+      { trainee_id: "C7947", trn: 2073497 },
+      { trainee_id: "C123420", trn: 2073540 },
+      { trainee_id: "1443603", trn: 2073562 },
+    ].each do |attrs|
+      trainee = Trainee.find_by(trainee_id: attrs[:trainee_id])
```

Comment:
I don't think `trainee_id` is necessarily unique across all trainees.

Reply:
True, but I checked on prod and there is only one record for each trainee ID. But going forward, we should request more information to ensure uniqueness.